src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.model;
+
+import racingcar.strategy.MovingStrategy;
+import racingcar.strategy.impl.RandomBoundMovingStrategy;
+import racingcar.util.RandomNumberUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private final List<Car> cars;
+
+    private Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public static Cars of(List<Car> cars) {
+        return new Cars(cars);
+    }
+
+    public static Cars of(int numberOfCar, MovingStrategy movingStrategy) {
+        List<Car> cars = new ArrayList<Car>();
+
+        for (int i = 0; i < numberOfCar; i++) {
+            Car car = new Car();
+            car.setMovingStrategy(movingStrategy);
+            cars.add(car);
+        }
+        return Cars.of(cars);
+    }
+
+    public int getCarsCount() {
+        return cars.size();
+    }
+
+    public List<Integer> getCarsPositions() {
+        return cars.stream().map(car -> car.getPosition()).collect(Collectors.toList());
+    }
```

Comment:
```suggestion
    public List<Integer> getCarsPositions() {
        return cars.stream()
            .map(car -> car.getPosition())
            .collect(Collectors.toList());
    }
```
이렇게 스트림 메소드의 경우 한 줄에 한 개만 사용해주세요!

Reply:
넵 다음 step 진행 하기 전에 반영 하겠습니다 !

Reply:
넵 바로 적용하겠습니다! 다음 step 에서도 유의하여 사용 하겠습니다 ㅎㅎ